STYLE-GUIDE.md

```diff
@@ -2,38 +2,36 @@
 
 This document defines some general styles we adhere to in the docs.
 
-## Words are important
+## Use Inclusive Language
 
-- Instead of _guys_ try _folks_, _yall_, or _everyone_
-- Instead of _crazy_ try _wild_
-- Instead of _click_ use _select_
-- Instead of _dummy_ use _placeholder_
-- Avoid using gendered language
-- Avoid using violent language (e.g., _kill_)
-- Avoid pop-culture references
+* Instead of _crazy_ try _wild_
+* Instead of _click_ use _select_
+* Instead of _dummy_ use _placeholder_
+* Avoid unnecessarily gendered language: Instead of _guys_ try _folks_, _yall_, or _everyone_.
+* Avoid using violent language (e.g., _kill_)
+* Avoid pop-culture references as such references may not be familiar to all readers.
 
 Want to learn about why these words should be avoided?
 Check out [Google's inclusive documentation guide](https://developers.google.com/style/inclusive-documentation).
 
 ## Headings
 
-- You should be able to scan the headings of a document and get a good sense of the page
-- Every page should have exactly one h1
-- Don't skip heading levels in Markdown files, e.g., if your previous heading level was a 2, the next heading should be a 2 or a 3, but not a 4 or 5
+* Readers should be able to scan the headings of a page and get an effective overview of the page's content.
```

Comment:
i prefer this not be put on the readers and instead on the authors of the docs... can you say more about why _you_ is bad to use here?

Reply:
I changed it as a stylistic thing - I was kinda trained not to use second-person singular in order to make the language more neutral, but my style may be outdated here, e.g.: https://docs.microsoft.com/en-us/style-guide/grammar/person

Let me do a little more research into what the Big Tech Companies recommend style-wise.

Reply:
okay id prefer to keep it in this situation because we are describing things we want the docs authors to do. we could adjust this to _as an author_... if you prefer though. i don't want it to read as though its on the readers though.